src/app/dashboard/dashboard.html

```diff
@@ -1,33 +1,26 @@
+<<<<<<< HEAD
+<div id="dashboardContainer">
+
+	<header id="dashboardHeader">
+			<nav>
+				<ul>
+					<li>
+						<img src="" />
+						<a href="http://localhost:8080/#/">Log Out</a><!-- http://localhost:8080/#/ -->
+					</li>
+				</ul>
+=======
```

Comment:
you can delete lines 12-19 and line 1
you will have to move the ng-click matt wrote and merge it with your link to have a final result of
`<a href="" href="" ng-click="dash.logout()">Log Out</a>`

at the end your file should look like this
```
<div id="dashboardContainer">
  <header id="dashboardHeader">
    <nav>
      <ul>
        <li>
          <img src="" />
          <a href="http://localhost:8080/#/">Log Out</a><!-- http://localhost:8080/#/ -->
        </li>
      </ul>
    </nav>
  </header>
  <div id="tray" ui-view></div>
  <footer>
    <!-- NOTHING HERE YET -->	
  </footer>
</div>
```